test/e2e/nodenetworkstate_test.go

```diff
@@ -0,0 +1,56 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+)
+
+var _ = Describe("NodeNetworkState", func() {
+	Context("when desiredState is configured", func() {
+		var (
+			desiredState nmstatev1.State
```

Comment:
please do it on a single line, save trees

Reply:
Done